js/backbone.upload-manager.js

```diff
@@ -465,7 +478,15 @@
             computeData: function ()
             {
                 return $.extend(this.getHelpers(), this.model.get('data'));
-            }
+            },
+            
+            // Max Wong: Cannot add same file after cancelled #21
+            _clearPreviousAddFileInput: function() {
```

Comment:
Um... I think I am to green to refactor it.
I dont know how to make it across 2 objects.